test/import.js

```diff
@@ -44,17 +46,42 @@ exports.import = {
         test.done();
     },
 
-    "supports a Date instance": function(test) {
+    "supports a Date instance greater than 5 days old": function(test) {
         var event = "test",
-            time = new Date,
+            six_days_ago = mock_now_time - 1000 * 60 * 60 * 24 * 6,
+            time = new Date(six_days_ago),
             props = { key1: 'val1' },
             expected_endpoint = "/import",
             expected_data = {
                 event: 'test',
                 properties: {
                     key1: 'val1',
                     token: 'token',
-                    time: 0
+                    time: six_days_ago / 1000
+                }
+            };
+
+        this.mixpanel.import(event, time, props);
+
+        test.ok(
+            this.mixpanel.send_request.calledWithMatch(expected_endpoint, expected_data),
+            "import didn't call send_request with correct arguments"
+        );
+
+        test.done();
+    },
+
+    "supports a Date instance less than 5 days old": function(test) {
+        var event = "test",
+            time = new Date(mock_now_time),
+            props = { key1: 'val1' },
+            expected_endpoint = "/track",
```

Comment:
@tdumitrescu, it is expected that the `/track` rather than `/import` endpoint will always be used if the event is less than five days old, correct?